.github/workflows/debug-kindle-link.yml

```
@@ -0,0 +1,38 @@
+    steps:
+      - uses: actions/checkout@v4
+      - name: Cache Cargo
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            backend/target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('backend/Cargo.lock') }}
+      - name: Install Rust 1.97.1
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.12)

[error] the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)